ci/build-common.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+common_args="--werror \
+-Dlibmpv=true \
+-Dtests=true \
+"
+
+export CFLAGS="$CFLAGS -Wno-error=deprecated -Wno-error=deprecated-declarations -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3"
```

Comment:
Why `-U_FORTIFY_SOURCE`? Something already sets that?

Reply:
It's already set in the alphin container. We could avoid this by omitting the leading `$CFLAGS` there. Not sure if we wanted to or not though.